[Docs] Add note about TSVB axis scaling (#91211)
elastic/kibana: docs/user/dashboard/tsvb.asciidoc

==== Visualization options

Time series::

>   The Y axis defaults to showing the full range of data, including zero. To scale the axis from

Reviewer: ```suggestion
  By default, the y-axis shows the full range of data, including zero. To scale the axis from
```

  the minimum to maximum values of the data automatically, go to *Series > Options > Fill* and set *Fill to 0*.

>   Annotations can be added to the X axis from based on timestamped documents in a separate {es} index.

Reviewer: ```suggestion
  You can add annotations to the x-axis based on timestamped documents in a separate {es} index.
```
